include/nvs/nvs.h

```diff
@@ -91,16 +91,17 @@ struct nvs_fs {
   */
 int nvs_init(struct nvs_fs *fs, const char *dev_name, u32_t magic);
 
-/**
- * @brief nvs_clear
- *
- * Clears the NVS file system from flash.
- *
- * @param fs Pointer to file system
- * @retval 0 Success
- * @retval -ERRNO errno code if error
- */
-int nvs_clear(struct nvs_fs *fs);
```

Comment:
I don't know zephyr enough to know about the API policy, but removing this function is breaking the API. I know there is also a file system format change, so that might be fine after all.

Reply:
I will reintroduce nvs_clear, if the user would like to clear the flash area for some other purpose, nvs_clear should be there.